clouddriver-artifacts/src/test/java/com/netflix/spinnaker/clouddriver/artifacts/helm/IndexParserTest.java

```diff
@@ -0,0 +1,78 @@
+package com.netflix.spinnaker.clouddriver.artifacts.helm;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class IndexParserTest {
+	
+	private String buildIndexYaml(String chartName, String version, List<String> urls) {
+		StringBuilder indexYamlBuilder = new StringBuilder();
+		indexYamlBuilder
```

Comment:
Might have been a bit easier to read to just construct as an object, then convert to yaml at the end